lib/livebook/intellisense/signature_matcher.ex

```diff
@@ -19,11 +19,12 @@ defmodule Livebook.Intellisense.SignatureMatcher do
           {:ok, list(signature_info()), active_argument :: non_neg_integer()} | :error
   def get_matching_signatures(hint, intellisense_context) do
     %{env: env} = intellisense_context
+    node = intellisense_context.node
```

Comment:
We need to pass the node as an option here too. `intellisense_context` comes from the node that does evaluation, and for the remote cell that's not the node we want to use.